src/Tasks/Resources/Strings.resx

```diff
@@ -931,7 +930,6 @@
   </data>
   <data name="GenerateResource.DeleteCorruptOutputFailed">
     <value>MSB3560: Could not delete the possibly corrupt output file "{0}". {1}</value>
-    <comment>{StrBegin="MSB3560: "}</comment>
```

Comment:
If we leave this as a warning we can revert this change.

Reply:
okay, so just add back in the line `<comment>{StrBegin="MSB3560: "}</comment>` below 932?

Reply:
Yes.

Reply:
*should* be fixed